### ISSUE TITLE
Conduct a11y audit of ILN

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon-addons-cloud#16]
We've gotten some feedback from our internal a11y experts that ILN is failing basic a11y requirements. This issue is to do the following:

- [ ] Conduct the initial a11y audit to find initial problems with the component
  - [ ] Verify with an a11y expert
- [ ] Perform initial implementation of fixes for the a11y violations
- [ ] Verify update actually addresses a11y violations reported in the initial audit

**Comment by marijohannessen** [comment]
Tracking this in https://github.com/carbon-design-system/carbon-addons-cloud/issues/26
